packages/kbn-mock-idp-plugin/server/plugin.ts

```diff
@@ -18,10 +25,30 @@ const createSAMLResponseSchema = schema.object({
   roles: schema.arrayOf(schema.string()),
 });
 
+const projectToAlias = new Map<string, string>([
+  ['observability', 'oblt'],
+  ['security', 'security'],
+  ['search', 'es'],
+]);
```

Comment:
I think we need to have a better place for it: our scripts & FTR use aliases and kibana.yml use actual project types, and it is not always obvious what `projectType` actually means, 'oblt' or 'observability'. WDYT?

Reply:
I'm in favor of skipping the mapping and using the exact project type.  With that said, if this is the only place we need it now we could leave it and refactor later.  Thoughts?

Reply:
>I'm in favor of skipping the mapping and using the exact project type

Agree with @jbudz on using the exact project types. We can probably open the issue to update scripts cli & kbn-es/kbn-test, not sure about other areas (e.g. Cypress test runner)